### PR TITLE
Adding loki to doco

### PR DESCRIPTION
**Pull request by joshuafernandes**
# Pull request checklist

Use the following template to make sure your PR fits the GoQuorum documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-locally).

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools (e.g., lint)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [ ] You've [previewed your changes on GitHub Pages](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-on-github-pages)

### CODE REVIEW

File: docs/configure-and-manage/monitor/loki.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+[Grafana Loki] is an open-source log management platform that is available when using the
+[Quorum Developer Quickstart](../../tutorials/quorum-dev-quickstart/index.md).
+
+The [Promtail configuration] ingests logs at regular defined intervals and outputs them to [Loki] for storage.
+
+The `pipeline configuration` in Promtail defines pipeline stages that can collate logs natively or using the JSON format.
+
+!!! note
+
+    If using the pipeline regex stage in `Promtail`, configuration must match the your log format.
```
Comment (rolandtyler) [inline docs/configure-and-manage/monitor/loki.md:16]:
```suggestion
    If using the pipeline regex stage in `Promtail`, configuration must match your log format.
```

File: docs/configure-and-manage/monitor/loki.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+1. Open the [`Grafana Loki address`](http://localhost:3000/d/Ak6eXLsPxFemKYKEXfcH/quorum-logs-loki?orgId=1&var-app=quorum&var-search=&from=now-15m&to=now) listed
+    by the sample networks `list.sh` script.
+
+    The logs display in Loki.
+
+    ![Loki logs](../../images/dashboard_grafana_loki.png)
+
+<!-- Links -->
+[Promtail configuration]: https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/common/promtail/promtail.yml
+[pipeline configuration]: https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/common/promtail/promtail.yml
```
Comment (rolandtyler) [inline docs/configure-and-manage/monitor/loki.md:31]: \[pipeline configuration\] wasn't used as a link, so you can delete this line or add the link to line 12 to clear the lint error.

**Review by rolandtyler — APPROVED** [review]
Minor comments, and recommend adding Nico's SEO suggestions from https://github.com/hyperledger/besu-docs/pull/1152, otherwise LGTM

File: docs/configure-and-manage/monitor/loki.md (hunk 2 above)
Comment (joshuafernandes) [reply docs/configure-and-manage/monitor/loki.md]: :+1: done

**Pull request merged**